tests/sparsezoo/analysis/test_helpers.py

```diff
@@ -0,0 +1,368 @@
+    node = get_node_from_name(model, node_name)
+
+    assert get_node_sparsity_sizes(model, node) == (
+        expected_num_zeros,
+        expected_param_size,
+    )
+
+
+@pytest.mark.parametrize(
+    "model_name,node_name,expected_num_zero_blocks,expected_num_blocks",
+    [
+        ("mobilenet_v1_pruned_moderate", "Conv_72", 87754, 131072),
+        ("mobilenet_v1_pruned_moderate", "BatchNormalization_79", 0, 0),
+        ("mobilenet_v1_pruned_moderate", "Unsqueeze_87", 0, 0),
+        ("mobilenet_v1_pruned_moderate", "Gemm_90", 0, 256000),
+        ("bert_pruned_quantized", "Gather_34", 0, 0),
+        ("bert_pruned_quantized", "DequantizeLinear_27", 0, 0),
+        ("bert_pruned_quantized", "MatMul_80_quant", 117964, 147456),
+        ("bert_pruned_quantized", "MatMul_157_quant", 0, 0),
+    ],
+)
+def test_get_node_four_block_sparsity_sizes(
+    model_name, node_name, expected_num_zero_blocks, expected_num_blocks
+):
+    model = get_model_from_name(model_name)
+    node = get_node_from_name(model, node_name)
+
+    assert get_node_four_block_sparsity_sizes(model, node) == (
+        expected_num_zero_blocks,
+        expected_num_blocks,
+    )
+
+
+@pytest.mark.parametrize(
+    "model_name,node_name,expected_value",
+    [
+        ("mobilenet_v1_pruned_moderate", "Conv_72", 0),
+        ("mobilenet_v1_pruned_moderate", "BatchNormalization_79", 0),
+        ("mobilenet_v1_pruned_moderate", "Unsqueeze_87", 0),
+        ("mobilenet_v1_pruned_moderate", "Gemm_90", 0),
+        ("bert_pruned_quantized", "Gather_34", 0),
+        ("bert_pruned_quantized", "DequantizeLinear_27", 0),
+        ("bert_pruned_quantized", "MatMul_80_quant", 128),  # MatMulInteger
+        ("bert_pruned_quantized", "MatMul_157_quant", 0),  # QLinear
+    ],
+)
+def test_get_zero_point(model_name, node_name, expected_value):
+    model = get_model_from_name(model_name)
+    node = get_node_from_name(model, node_name)
+
+    assert get_zero_point(model, node) == expected_value
+
+
+@pytest.mark.parametrize(
+    "model_name,node_name,expected_value",
+    [
+        ("yolact_none", "Conv_0", 0),
+        ("yolact_none", "Conv_33", 0),
+        ("yolact_none", "LeakyRelu_36", 0),
+        ("yolact_none", "Conv_275", 0),
+        ("mobilenet_v1_pruned_moderate", "Conv_72", 0.8999996185302734),
+        ("mobilenet_v1_pruned_moderate", "BatchNormalization_79", 0),
+        ("mobilenet_v1_pruned_moderate", "Unsqueeze_87", 0),
+        ("mobilenet_v1_pruned_moderate", "Gemm_90", 0),
+        ("bert_pruned_quantized", "Gather_34", 0),
+        ("bert_pruned_quantized", "DequantizeLinear_27", 0),
+        ("bert_pruned_quantized", "MatMul_80_quant", 0.8029022216796875),
+        ("bert_pruned_quantized", "MatMul_157_quant", 0),
+    ],
+)
+def test_get_node_sparsity(model_name, node_name, expected_value):
+    model = get_model_from_name(model_name)
+    node = get_node_from_name(model, node_name)
+
+    assert get_node_sparsity(model, node) == expected_value
```

Comment:
we can give a margin of error to this check to potentially avoid failing on minor system rounding errors (ie `assert abs(sparsity - expected_sparsity) <= 1e-5`)